scripts/remove-extraneous-types.cjs

```diff
@@ -0,0 +1,13 @@
+/**
+ * Removes extraneous types left after combining types into a single file using rollup-plugin-dts
+ */
+const rimraf = require("rimraf");
+const path = require("path");
+
+const foldersToRemove = ["../lib/esm/types-tsc", "../lib/umd/types-tsc"];
+
+foldersToRemove.forEach((dir) => {
+  const fullPath = path.resolve(__dirname, dir);
+  rimraf.sync(fullPath);
+  console.log(`Removed: ${fullPath}`);
+});
```

Comment:
This script can be deleted in favor of using rimraf cli

```suggestion
```

Reply:
Replaced with direct rimraf CLI usage. Typically, I like to have it as a separate script, just in case some additional cleanup is needed in the future, and for logging.